guides/source/active_support_instrumentation.md

```diff
@@ -226,6 +226,20 @@ Action View
 }
 ```
 
+### render_collection.action_view
+
+| Key           | Value                 |
+| ------------- | --------------------- |
+| `:identifier` | Full path to template |
+| `:count`      | Size of collection    |
+
+```ruby
+{
+  identifier: "/Users/adam/projects/notifications/app/views/posts/index.html.erb",
```

Comment:
This is still a partial, so something like "/Users/adam/projects/notifications/app/views/posts/_comment.html.erb" would be a better name here.